### ISSUE TITLE
Pre-commit hook failing

### ISSUE THREAD

**Issue by adriantorrie** [markdownlint/markdownlint#389]
## Description

```sh
$ pre-commit run --all-files
[ERROR] `markdownlint_docker` is not present in repository https://github.com/markdownlint/markdownlint.  Typo? Perhaps it is introduced in a newer version?  Often `pre-commit autoupdate` fixes this.
```

```sh
$ pre-commit autoupdate
Updating https://github.com/markdownlint/markdownlint ... Cannot update because the update target is missing these hooks:
markdownlint_docker
```

My settings

```yaml
  # Markdown linter
  - repo: https://github.com/markdownlint/markdownlint
    rev: v0.11.0
    hooks:
      - id: markdownlint_docker
        language: docker_image
        files: \.(md|mdown|markdown)$
        entry: markdownlint/markdownlint
```

### Environment

**MDL Version**

### Expected Behavior
- Pre-commit hook should be installed

### Actual Behavior
- Pre-commit hook doesn't have the hook named `markdownlint_docker` available
- Pre-commit hook not installed

**Comment by s1e2b3i4** [comment]
This happens because the released version in [v0.11.0](https://github.com/markdownlint/markdownlint/blob/v0.11.0/.pre-commit-hooks.yaml) does not contain the `markdownlint_docker` hook.
To use this hook, you have to set the `rev:` to a commit which includes this change.
Instead, you can just use the most recent commit to the master branch [as of today]: 

```yaml
  - repo: https://github.com/markdownlint/markdownlint
    rev: 5329a84691ab0fbce873aa69bb5073a6f5f98bdb
    hooks:
      - id: markdownlint_docker
```
